File: app/src/main/java/com/startogamu/zikobot/core/module/spotify_auth/model/SpotifyRefreshToken.java
package com.startogamu.zikobot.core.module.spotify_auth.model;

import com.google.gson.annotations.SerializedName;

import lombok.AllArgsConstructor;
import lombok.Data;

@Data
@AllArgsConstructor(suppressConstructorProperties = true)
public class SpotifyRefreshToken {

    @SerializedName("grant_type")
    private final String grantType;
    @SerializedName("refresh_token")
    private final String refreshToken;
    @SerializedName("redirect_uri")
    private final String redirectUri;
    @SerializedName("client_id")
    private final String cliendId;
    @SerializedName("client_secret")
    private final String clientSecret;
}

File: data/src/main/java/com/joxad/zikobot/data/module/spotify_auth/model/SpotifyRefreshToken.kt
package com.joxad.zikobot.data.module.spotify_auth.model

import com.google.gson.annotations.SerializedName

class SpotifyRefreshToken(@SerializedName("client_secret") val clientSecret: String? = null, @SerializedName("client_id") val cliendId: String? = null, @SerializedName("grant_type") val grantType: String? = null, @SerializedName("refresh_token") val refreshToken: String? = null, @SerializedName("redirect_uri") val redirectUri: String? = null) {

}
